src/list-sorting/index.js

```diff
@@ -1,3 +1,14 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+  if (Array.isArray(haystack[0])) {
+    let result = [];
+    for (let i = 0; i < haystack.length; i++) {
+      if (haystack[i].includes(needle)) {
+        result.push(i, haystack[i].lastIndexOf(needle));
+      }
+    }
+    return result.length ? result.slice(result.length - 2) : -1;
```

Comment:
If you did a reverse loop, you could pick the first occurence you find and not have to do this.